### DISCUSSION TITLE
Is Ethernet enabled by default

### DISCUSSION THREAD

**Discussion by ritsp** [StealthChesnut/HA-FoxESS-Modbus#73] [Q&A] [answered]
Hello everyone.... hoping you can help a newbie. 
I have connected the ethernet cable to the inverter port, and to a switch that is connected to my router and internal network. So effectively on the same network as my PC, HOMEASSISTANT instance etc. 

No matter what i do i cannot get my inverter to pick up an IP address and connect to the network. 
DHCP is enabled but nothing i do works. 

Is the Ethernet port active, or to i need to do something to activate it ?
Inverter Model : H1-6.0-E
Inverter Hardware Version : -Master : 1.51 Slave : 1.02 Manager : 1.56

**Comment by andyhasalandy** [comment]
Have you rebooted the inverter since setting up and connecting to your LAN?

**Reply by ritsp** [reply]
i had tried this but no joy. thanks though. the version appeared to be the issue

**Comment by tagware** [comment, marked as answer]
It's the manager that is the issue.

V1.56 removes the Ethernet connection.

V1.50 will re in-state.

**Reply by ritsp** [reply]
perfect this solved the issue thank you. 
Asked my installer to "downgrade" to 1.50 (no idea how i would do this myself yet)

**Reply by bkortleven** [reply]
If you find out how/what to get the file: please let me/us know!!! 
You can update/downgrade via a USB storage key, check the inverter manual, the procedure is described in detail!

I have the exact same issue. I never asked for auto-updates, nor do I know how to disable them... and would like to downgrade to 1.5 and then immediately disable the default gateway setting...
